### DISCUSSION TITLE
Get Something Q&A 2

### DISCUSSION THREAD

**Discussion by quicksandzn** [Schweinepriester/github-profile-achievements#70] [Q&A] [answered]
Get Something Q&A 2

**Comment by quicksandznzn** [comment, marked as answer]
Q&A

**Comment by treeplate** [comment]
huh?

**Reply by elijahpereira** [reply]
he was getting a github achievement

**Reply by treeplate** [reply]
ah

**Comment by Veercodeprog** [comment]
You can take part in ongoing [coding problem](https://github.com/Veercodeprog/Data-Structures-implementation/discussions/categories/coding-discussion-share-your-solution-with-best-time-complexity-approach) discussion.

A daily problem will be posted, and the answer with the most efficient time complexity submitted first will be selected as the accepted solution.
